### PR TITLE
Changes PO description

### PR DESCRIPTION
**Pull request by Arctrooper8**
## About The Pull Request

Changes the PO duty and spawn message

## Why It's Good For The Game

The current PO duty and spawn message are ancient, the spawn message in particular suggests that you shouldn't leave the Almo without reason (POs haven't been flying the Alamo themselves way before the addition of automatic).
This updates them to more closely match modern policy and PO playstyle.

## Changelog
:cl:
qol: PO duty and spawn message have been updated
/:cl:

### CODE REVIEW

File: code/datums/jobs/job/shipside.dm [hunk 1]
```diff
@@ -326,9 +326,10 @@ You are in charge of logistics and the overwatch system. You are also in line to
 
 /datum/job/terragov/command/pilot/radio_help_message(mob/M)
 	. = ..()
-	to_chat(M, {"Your job is to fly, protect, and maintain the ship's dropship.
-While you are a warrant officer, your authority is limited to the dropship, where you have authority over the enlisted personnel.
-If you are not piloting, there is an autopilot fallback for command, but don't leave the dropship without reason."})
+	to_chat(M, {"Your job is to support marines with either close air support via the Condor, or mobile dropship support with the Tadpole.
+While you are in charge of all aerial crafts the Alamo does not supervision outside of turning automatic mode on or off at crucial times, and you are expected to choose between the Condor and Tadpole.
```
Comment (Mechanelize) [inline code/datums/jobs/job/shipside.dm:330]: typo

File: code/datums/jobs/job/shipside.dm [hunk 2]
```diff
@@ -290,7 +290,7 @@ You are in charge of logistics and the overwatch system. You are also in line to
 		<b>You answer to the</b> acting Command Staff<br /><br />
 		<b>Unlock Requirement</b>: Starting Role<br /><br />
 		<b>Gamemode Availability</b>: Distress<br /><br /><br />
-		<b>Duty</b>: Listen and coordinate your crew for the sole dropship you will use. Provide transport and close air support for the marines during the mission. Ensure no threats come aboard your dropship and by extension, the vessel which houses the crew.
+		<b>Duty</b>: Choose between the Condor: a modular attack aircraft that provides close air support with a variety of weapons ranging from the inbuilt gatling to wing mounted rockets; or the Tadpole, a versatile dropship capable of fulfilling roles ranging from ambulance to mobile bunker.
```
Comment (Wayland-Smithy) [inline code/datums/jobs/job/shipside.dm:293]: Just proofreading things
```suggestion
		<b>Duty</b>: Choose between the Condor, a modular attack aircraft that provides close air support with a variety of weapons ranging from the inbuilt gatling to wing mounted rockets; or the Tadpole, a versatile dropship capable of fulfilling roles ranging from ambulance to mobile bunker.
```

File: code/datums/jobs/job/shipside.dm (hunk 1 above)
Comment (Wayland-Smithy) [inline code/datums/jobs/job/shipside.dm:330]:
```suggestion
While you are in charge of all aerial crafts the Alamo does not require supervision outside of turning automatic mode on or off at crucial times, and you are expected to choose between the Condor and Tadpole.
```

Comment (Arctrooper8) [reply code/datums/jobs/job/shipside.dm:330]: Danke!

File: code/datums/jobs/job/shipside.dm (hunk 2 above)
Comment (Arctrooper8) [reply code/datums/jobs/job/shipside.dm:293]: Danke!

**Comment by Arctrooper8** [comment]
Grammar good, my thanks

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**